pkg/controller/bypass/bypass_controller.go

```diff
@@ -103,16 +117,24 @@ func StartByPassController(client kubernetes.Interface) error {
 				log.Debugf("%s/%s: Pod is being deleted, skipping further processing", pod.GetNamespace(), pod.GetName())
 				return
 			}
+
+			log.Infof("%s/%s: disable bypass control", pod.GetNamespace(), pod.GetName())
 			enableSidecar, _ := checkSidecar(client, pod)
-			if !enableSidecar {
-				log.Info("do not need delete iptables rules, pod is not managed by sidecar")
-				return
+			enableKmesh := isKmeshManaged(pod)
+
+			if enableSidecar {
+				nspath, _ := getnspath(pod)
+				if err := deleteIptables(nspath); err != nil {
+					log.Errorf("failed to add iptables rules for %s: %v", nspath, err)
+					return
+				}
 			}
-			log.Infof("%s/%s: DELETED", pod.GetNamespace(), pod.GetName())
-			nspath, _ := getnspath(pod)
-			if err := deleteIptables(nspath); err != nil {
-				log.Errorf("failed to delete iptables rules for %s: %v", nspath, err)
-				return
+			if enableKmesh {
+				nspath, _ := getnspath(pod)
+				if err := handleKmeshBypass(nspath, 932); err != nil {
+					log.Errorf("failed to enable kmesh control")
```

Comment:
The log information may cause misunderstanding. Pls update the log description.

Reply:
> The log information may cause misunderstanding. Pls update the log description.

modified